src/collective/casestudy/interfaces.py

```diff
@@ -14,15 +14,22 @@ class ICaseStudySettings(Interface):
     industries = schema.List(
         title=_("Industries."),
         description=_(
-            "help_industries", default="List of industries available for Case Studies"
+            "help_industries",
+            default=(
+                "List of industries available for Case Studies"
+                "Use the format key|value"
+            ),
         ),
         value_type=schema.TextLine(),
         default=[],
     )
 
     usages = schema.List(
         title=_("Usage."),
-        description=_("help_usages", default="Categories of Plone usage"),
+        description=_(
+            "help_usages",
+            default=("Categories of Plone usages. " "Use the format key|value"),
```

Comment:
Both "usage" and "usages" are correct forms in this context, as it can be "uncountable". I would use "usage".

"Usages" certainly does not roll off the tongue as elegantly as "einsatzbereich" 😜